dist/logzioBlobAzureFunction/core.js

```diff
@@ -47,30 +53,43 @@ function sendData(data, logType, context) {
   logzioShipper.sendAndClose(callBackFunction);
 }
 
-function getData(url, compressed, callback) {
+function extractFileType(url, isCompressed){
+  const urlArray = url.split(".");
+  if (isCompressed){
+    urlArray.pop();
+  }
+  const fileType  = urlArray.pop();
+  var format = process.env.Format;
```

Comment:
const

Reply:
This is the returned variable, it may get a different value in the cases below

Reply:
but the value won't change